### PR TITLE
Added support for more statements.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: src/preprocess.inl.js [hunk 1]
```diff
@@ -192,6 +304,8 @@ const wrap_for_assignment = (x) => {
 #define CallStmtBegin(func, target, rowcol) target = (() => { let args = [];
 #define CallStmtPassArg(arg_index, arg_value) args[arg_index] = arg_value;
 #define CallStmtEnd(func, target) return opa_get_function_impl(func)(args)})();
+#define NotStmtBegin(rolcol) if ((() => {
```
Comment (mzhurovich) [inline src/preprocess.inl.js:307]: `ro**w**col`?

File: tests/all_tests.js.regenerate.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ console.log('    } else {');
 console.log('      return exported_mains[`${fn}.js`];');
 console.log('    }');
 console.log('  } catch(e) {');
-console.log('    console.error("Error loading `./${fn}.js`, did you run `./scripts/gen_all_js.sh`?");');
+;
```
Comment (mzhurovich) [inline tests/all_tests.js.regenerate.js:27]: :-/

File: src/preprocess.inl.js (hunk 1 above)
Comment (dkorolev) [reply src/preprocess.inl.js]: :+1: e337711

File: tests/all_tests.js.regenerate.js (hunk 2 above)
Comment (dkorolev) [reply tests/all_tests.js.regenerate.js:27]: It didn't print the right thing before, apparently :-/

**Pull request merged**